frontend/src/metabase/selectors/settings.ts

```diff
@@ -26,6 +31,54 @@ export const getLearnUrl = (path = "") => {
   return `https://www.metabase.com/learn/${path}`;
 };
 
+interface DocsUrlProps {
+  page?: string;
+  anchor?: string;
+}
+
+export const getDocsUrl = createSelector(
+  (state: State) => getSetting(state, "version"),
+  (state: State, props: DocsUrlProps) => props.page,
+  (state: State, props: DocsUrlProps) => props.anchor,
+  (version, page, anchor) => getDocsUrlForVersion(version, page, anchor),
+);
+
+export const getDocsUrlForVersion = (
```

Comment:
nit: This isn't a selector, considering this file is `selectors/settings.ts`. I'm fine leaving it here since the purpose is to be used in `getDocsUrl`. However it's name looks very similar to `getDocsUrl` and might be mistaken for a selector since it's also exported. I get that it's used in `frontend/src/metabase/lib/settings.ts`.

What do you think about communicating that this isn't a selector? e.g. 
- Change the function name
- Add comments saying this is supposed to be private, but it's exported until `frontend/src/metabase/lib/settings.ts`'s  `docsUrl` is fixed.

Reply:
Agree, added a comment